service-manual/user-centred-design/resources/patterns/index.md

```diff
@@ -15,44 +15,44 @@ breadcrumbs:
 ---
 
 {:.intro}
-These patterns take the GOV.UK elements and combine them to meet common user needs.
-Use them to maintain a consistent design across our services.
+The Design Patterns section is split into two areas, broadly organised according to the main task they are supposed to help with. 
 
+<ul>
+ <li>If you're doing design then you'll probably find the Detail and Process patterns (below) most useful; they're mostly about which patterns to use and why we're recommending them.</li>
+ <li>If you're doing development then you'll probably find [the GOV.UK elements](/service-manual/user-centred-design/resources/elements)  most useful; they're mostly about how to make your service look consistent with GOV.UK and include links to code snippets. </li>
+</ul>
 
-### Forms
+Whatever you're doing, you'll need to use these patterns as part of a user-centred design approach.
 
-See [the GOV.UK elements page](/service-manual/user-centred-design/resources/elements)
```

Comment:
This link is appearing as `[the GOV.UK elements page](/service-manual/user-centred-design/resources/elements)` in the rendered page. I think it needs to be HTML, like those below that were deleted:

`<a href="/service-manual/user-centred-design/resources/elements">the GOV.UK elements</a>`